test/python/conftest.py

```diff
@@ -24,6 +24,7 @@
     "DOT": "Polkadot",
     "tron": "Tron",
     "ton": "TON",
+    "cosmos": "Cosmos",
```

Comment:
Needs to be aligned with the value in the CI
```suggestion
    "ATOM": "Cosmos",
```